### PR TITLE
Repeater tag for dynamic labelling

### PR DESCRIPTION
**Pull request by gfauchart**
Hello,
I've added the example so you can see how it would work!

Any feedback let me know!

### CODE REVIEW

**Comment by nicholasrq** [comment]
@gfauchart thank you, great work!

 please, add tests and provide some usage examples here in the PR. once it's done we can merge your work

**Comment by gfauchart** [comment]
For the use case you can use
```xml
   <Repeater on="$utterances">
      <View>
        <Text name="user_{{idx}}" value="$.utterances[{{idx}}].text" />
      </View>
      <View>
        <header value="Utterance Review"></header>
        <Choices name="utterance_action" showInline="true" toName="user_{{idx}}">
          <Choice value="No Action" selected="true" />
          <Choice value="Training" />
          <Choice value="New Intent" />
        </Choices>
      </View>
   </Repeater>
```
with this data
```json
        "data": {
            "utterances": [
                {"text": "hello, when do i receive the product I ordered?"},
                {"text": "nevermind. I just receveid it"}
            ]
        }
```
image

**Review by makseq — APPROVED** [review]

**Review by nicholasrq — APPROVED** [review]

**Comment by makseq** [comment]
Hi, @gfauchart! Could you please commit again? Just some empty diff with spaces. We are trying to run github tests for forked PRs :slightly_smiling_face:

**Review by makseq — CHANGES_REQUESTED** [review]
Commit spaces, please :-)

**Review by makseq — APPROVED** [review]

File: src/core/Tree.js [hunk 1]
```diff
@@ -164,6 +165,29 @@ function treeToModel(html) {
     return hypertexts[htseen];
   }
 
+  function cloneXmlTreeAndReplaceKeys(root, idx, indexFlag = "{{idx}}") {
+    function recurseiveClone(node) {
```
Comment (nicholasrq) [inline src/core/Tree.js:169]: typo, must be `recursive`

File: src/utils/data.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { isString, escapeHtml } from "./utilities";
+import jsonpath from 'jsonpath';
```
Comment (nicholasrq) [inline src/utils/data.js:2]: do we really need extra library for that? is there an alternative maybe?
it's 403 kb unpacked and kinda heavy for one place usage

**Review by nicholasrq — CHANGES_REQUESTED** [review]

File: src/utils/data.js (hunk 2 above)
Comment (gfauchart) [reply src/utils/data.js]: I've changed to use lodash.get, I would imagine the size would be smaller

**Review by nicholasrq — APPROVED** [review]

**Pull request merged**